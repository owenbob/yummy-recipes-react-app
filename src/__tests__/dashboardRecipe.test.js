import React from 'react';
import ReactDOM from 'react-dom';
import { render,  configure,shallow } from 'enzyme';
import {shallowToJson} from 'enzyme-to-json';
import expect from "expect";
import Adapter from 'enzyme-adapter-react-16';
import DashboardRecipes from '../components/dashboardRecipes';
import LocalStorageMock from '../setupTests';
import MuiThemeProvider from 'material-ui/styles/MuiThemeProvider';


// configure({ adapter: new Adapter() });

let match = {
    params : '1'
}

const setUp = () => {
    const props = {
        match
    };
    return shallow(<DashboardRecipes {...props} />);
};


describe('DashboardRecipes', () => {
    const props ={
        history:{
            push: () => {}
        }
    }
    it('matches  snapshot', () => {
            const dashboardRecipes = setUp;
           expect(shallowToJson(dashboardRecipes)).toMatchSnapshot();
        });

    it('renders without crashing', () => {
            const div = document.createElement('div');
            ReactDOM.render(
            <MuiThemeProvider>
            <DashboardRecipes {...props} />
            </MuiThemeProvider>, div);
    });
    
})