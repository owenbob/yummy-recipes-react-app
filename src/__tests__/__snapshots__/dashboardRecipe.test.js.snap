// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DashboardRecipes matches  snapshot 1`] = `null`;
